### PR TITLE
Fix issues with config file and URLTweet announcer

### PR DESCRIPTION
**Pull request by itpun**
This is in response to issue #57 .

This will update the conf.example file to include everything needed for developers to get an idea what needs to be included to test out all features of Vilebot. I've also fixed an issue with URLTweet announcer so that it doesn't constantly spam the error message if you are sending messages while Vilebot is in the room and you have no set up the Twitter API keys. API keys should be optional and really should be modular so that they are only needed in your config file if you wanted to test/develop said feature.

### CODE REVIEW

File: vilebot/cfg/vilebot.conf.example [hunk 1]
```diff
@@ -13,7 +13,25 @@ ircPort1=6697
...
+accessToken=
+accessTokenSecret=
+
+# Retrieve from https://words.bighugelabs.com/getkey.php 
+# for jaziz intergration.
+thesaurusKey=
+
+# Retrieve from https://developer.wolframalpha.com/portal/apisignup.html 
+# for answerQuestion intergration.
+thesaurusKey=
```
Comment (yuqi-zhang) [inline vilebot/cfg/vilebot.conf.example]: Seems to be a duplicate

File: vilebot/cfg/vilebot.conf.example [hunk 2]
```diff
@@ -13,7 +13,25 @@ ircPort1=6697
 ircSslMode1=ON_NOHOST
 ircChannel1=#examplebot
 ircChannelAutoOp1=false
+
+# Retrieve these sections from https://apps.twitter.com
```
Comment (yuqi-zhang) [inline vilebot/cfg/vilebot.conf.example]: This and some other lines have whitespace

**Comment by lberk** [comment]
Where was the test to go along with this fix?

**Comment by itpun** [comment]
@lberk funny enough I made a mistake in merging this immediately I thought it was only going to merge to my forked one. Although I don't have a unit test to go with this, @yuqi-zhang was my QA for this patch. For the time being in order to get others interested in working with Vilebot I think this will be good enough.

**Pull request merged**